front/js/shared/tracking.es6

```diff
@@ -78,6 +67,44 @@
       });
   }
 
+  // @todo #759:30m Move PublishedGATracker to a separate file.
+
+  class PublishedGATracker {
+    constructor() {
+      this.published = false;
+    }
+
+    purchase(productsData, txData) {
```

Comment:
maybe it's possible to add tests for ecommerce behaviour.
We are already testing Ya.Metrika at SE: `shopelectro.tests.tests_selenium.YandexMetrika`